src/index.ts

```diff
@@ -20,8 +24,14 @@ const run = async () => {
     jsonSummary, jsonFinal 
   });
 
+  // Indicate working directory (if set) in Action summary for easier understanding (ie. in a monorepo)
+  let summaryHeading = "Coverage Summary";
+  if (workingDirectory) {
+    summaryHeading += ` for \`${workingDirectory}\``;
```

Comment:
Note that I have _no idea_ if this will actually work (saw it in another repo, but it was using a templating approach...), but there should be a way to include the working directory in the summary title (and formatted).

Reply:
I don't see why this wouldn't work.
I decided against a templating engine as I want as little dependencies as possible and so far, this action was simple enough to go with template strings.

Might change in the future, but for now this is the most simple approach imho. 🙂

Reply:
Yeah, I believe that Actions support markdown (for code ticks) in descriptions/comments, but never actually tested it 🤷